apps/portal/src/app/dashboard-component/dashboard.component.ts

```diff
@@ -23,13 +24,14 @@ import { PanelModule } from 'primeng/panel';
     templateUrl: './dashboard.component.html',
 })
 export class DashboardComponent {
-    private readonly functionsApi = inject(FunctionsApi);
+    private readonly functionsApi = inject(FirebaseFunctionsService);
 
     readonly data$ = this.functionsApi
         .call<{
             classes: Array<{ title: string; enrolledCount: string }>;
         }>('classes')
         .pipe(
+            RequestedOperatorsUtility.ignoreAllStatesButLoaded(),
```

Comment:
Temporarily migrated all usages of `FunctionsApi.call` to backwards-compatibly ignore new request states.